package.json

```diff
@@ -35,24 +35,25 @@
     "apollo-cache-inmemory": "^1.1.11",
     "apollo-client": "^2.2.7",
     "apollo-link": "^1.2.2",
+    "apollo-link-context": "^1.0.8",
     "apollo-link-error": "^1.0.9",
     "apollo-link-http": "^1.5.4",
     "body-parser": "^1.18.2",
     "chalk": "^2.3.2",
     "classnames": "^2.2.5",
+    "cookie-parser": "^1.4.3",
     "express": "^4.16.3",
     "graphql": "^0.13.2",
     "graphql-tag": "^2.8.0",
     "isomorphic-fetch": "^2.2.1",
     "js-cookie": "^2.2.0",
+    "keycloak-js": "4.0.0-beta.1",
```

Comment:
Based on the Keycloak [docs](https://www.keycloak.org/docs/4.0/securing_apps/#_javascript_adapter):

> A best practice is to load the JavaScript adapter directly from Keycloak Server as it will automatically be updated when you upgrade the server. If you copy the adapter to your web application instead, make sure you upgrade the adapter only after you have upgraded the server.

it'll be better if we're dependent on the Keycloak server for this lib. Not sure if that's an easy thing to do here

Reply:
@seun I understand why they recommend this on the client side and agree. What about the NodeJS server side? Are we using Keycloak there?

Reply:
A quick peek at the code setup shows that we `require` the keycloak lib only once in this [file](https://github.com/reactioncommerce/reaction-next-starterkit/blob/pr103-keycloak-integration/src/lib/auth/index.js). 

The comment in that file `// trigger oauth login redirect flow with Keycloak (CLIENT SIDE ONLY)` seem to imply that it's only run on the client side.

I'll dbl check to be sure.

Reply:
@impactmass Yes, seems to be client only. So, does that mean we should add a <script> element pointing to some `.js` file path on our own keycloak URL? I'm fine with that approach.

Reply:
@aldeed Yes. I'm doing that right now actually.